requirements-dev.txt

```diff
@@ -65,19 +66,17 @@ fonttools==4.53.0         # via -c constraints.txt, matplotlib
 fparser==0.1.4            # via -c constraints.txt, dace
 frozendict==2.4.4         # via -c constraints.txt, gt4py (pyproject.toml)
 gridtools-cpp==2.3.4      # via -c constraints.txt, gt4py (pyproject.toml)
-hypothesis==6.104.1       # via -c constraints.txt, -r requirements-dev.in, gt4py (pyproject.toml)
+hypothesis==6.104.2       # via -c constraints.txt, -r requirements-dev.in, gt4py (pyproject.toml)
 identify==2.5.36          # via -c constraints.txt, pre-commit
 idna==3.7                 # via -c constraints.txt, requests
 imagesize==1.4.1          # via -c constraints.txt, sphinx
-importlib-metadata==8.0.0  # via -c constraints.txt, build, jax, jupyter-client, sphinx
+importlib-metadata==8.0.0  # via -c constraints.txt, build, jupyter-client, sphinx
 importlib-resources==6.4.0 ; python_version < "3.9"  # via -c constraints.txt, gt4py (pyproject.toml), jsonschema, jsonschema-specifications, matplotlib
 inflection==0.5.1         # via -c constraints.txt, pytest-factoryboy
 iniconfig==2.0.0          # via -c constraints.txt, pytest
-ipykernel==6.29.4         # via -c constraints.txt, nbmake
+ipykernel==6.29.5         # via -c constraints.txt, nbmake
 ipython==8.12.3           # via -c constraints.txt, ipykernel
 isort==5.13.2             # via -c constraints.txt, -r requirements-dev.in
-jax[cpu]==0.4.13          # via -c constraints.txt, gt4py (pyproject.toml)
```

Comment:
We are removing jax from requirements-dev.txt because it's build for python 3.8. For me that's fine as we anyway don't have jax developers atm. Once we drop 3.8 and 3.9 we will have it back. What do you think @egparedes ?